test/externalTests/colony.sh

```diff
@@ -33,12 +33,12 @@ function colony_test
     OPTIMIZER_LEVEL=3
     CONFIG="truffle.js"
 
-    truffle_setup "$SOLJSON" https://github.com/solidity-external-tests/colonyNetwork.git develop_070_new
+    truffle_setup "$SOLJSON" https://github.com/solidity-external-tests/colonyNetwork.git develop_080_new
```

Comment:
Why do we have ``_new``-named branches there :-)?

Reply:
Ah right... https://github.com/ethereum/solidity/pull/10500 kind of explains it :-).

Reply:
Because versions without `_new` already exist and I didn't want to update them yet because that would break tests on `develop`. I put that change in a separate PR for that reason: #10500.

In `solidity-external-tests` I brought all the repos up to date with the upstream which changes the workarounds they require (different node.js version, forcing/not forcing Truffle version, patching `package.json`). #10429 takes care of that. It would actually be better to start reviewing with that one since it has all the context you need to understand what happened.